ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -81,11 +81,11 @@ fi
 
 # multi-instance setup
 # shellcheck disable=SC2154
-param set MAV_SYS_ID $((1+px4_instance))
```

Comment:
You cannot remove the +1 here, it breaks a lot of things.
If really needed, we can do it, but we need to change more than that.
Did you try what I suggested before in the launch script?
```
<arg name="tgt_system" value="$(eval 1 + arg('ID'))"/>
```